CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+- Updated read-only metastore whitelist environment variable name.
```

Comment:
Worth mentioning that this will work with `apiary-metastore-docker > v1.0.1`?

Reply:
apiary-metastore-docker version is yet to be created, we can update this when releasing new version.